src/EventStore.ClientAPI/CatchUpSubscriptionSettings.cs

```diff
@@ -3,37 +3,38 @@
 namespace EventStore.ClientAPI
 {
     /// <summary>
-    /// Settings for <see cref="EventStoreCatchUpSubscription"/>
+    /// Settings for <see cref="EventStoreCatchUpSubscription"/>.
     /// </summary>
     public class CatchUpSubscriptionSettings
     {
+        /// TODO: In what size?
```

Comment:
The unit here is the number of events in the queue, the same goes for the other TODO further on.

Reply:
@hayley-jean Updated :)